posthog/src/main/java/com/posthog/internal/errortracking/ThrowableCoercer.kt

```diff
@@ -78,6 +78,18 @@ public class ThrowableCoercer {
                     val fileName = frame.fileName
                     if (fileName?.isNotEmpty() == true) {
                         myFrame["filename"] = fileName
+                    } else {
+                        // If the file name is not available (e.g., the application has been compiled without debug info),
+                        // substitute with class and method names.
+                        val frameClass = frame.className?.takeIf { it.isNotEmpty() }
+                        val frameMethod = frame.methodName?.takeIf { it.isNotEmpty() }
+                        val fullyQualifiedName =
+                            listOfNotNull(frameClass, frameMethod)
+                                .takeIf { it.isNotEmpty() }
+                                ?.joinToString(".")
+                        if (fullyQualifiedName?.isNotEmpty() == true) {
+                            myFrame["filename"] = fullyQualifiedName
+                        }
```

Comment:
i think this should be a fallback on the frontend
also reported here https://posthog.slack.com/archives/C07AA937K9A/p1762352192041439?thread_ts=1762349699.084109&cid=C07AA937K9A
a reason would be that if the frame is  minified, it'd need to be symbolicated
another reason is that package+className != file name since they dont necessarily match, so its semantically wrong